python/cuml/cuml/tests/test_doctest.py

```diff
@@ -130,4 +130,9 @@ def test_docstring(docstring):
             in doctest_stdout.getvalue()
         ):
             pytest.skip("Doctest requires MG support.")
+        if (
+            "FutureWarning: In RAPIDS 25.06, "
```

Comment:
This future warning is about us changing the shape for release 25.06, should we do that change in this PR as well? Mainly would be moving `experimental/fil` to `fil` if I'm not mistaken

Reply:
The shape has already been changed

Reply:
I think we should remove the warning then.

Reply:
The warning is now removed.